CHANGES.rst

```diff
@@ -307,6 +307,10 @@ astropy.units
   of quantities. It produces unadorned or LaTeX strings, and accepts two different
   sets of delimiters in the latter case: ``inline`` and ``display``. [#8313]
 
+- Ensure classes that mimic quantities by having a ``unit`` attribute and/or
+  ``to`` and ``to_value`` methods can be properly used to initialize ``Quantity``
+   or set ``Quantity`` instance items. [#8535]
```

Comment:
There is an extraneous space at the start of line 312 that is causing the docs build to fail. Picky, eh?

Reply:
OK, thanks for noticing that. Now fixed...